eng/Packages.props

```diff
@@ -24,6 +24,7 @@
     <PackageReference Update="System.CodeDom" Version="4.4.0" />
     <PackageReference Update="System.Collections.Concurrent" Version="4.3.0" />
     <PackageReference Update="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Update="System.Configuration.ConfigurationManager" Version="4.5.0" />
```

Comment:
I think I asked this in a different PR: any concerns with DLL issues with VS insertions?

Reply:
It's referenced only for the Core flavor of MSBuild, so I think this is safe.

Reply:
Why did you need ConfigurationManger?  We don't reccomend that folks actually use ConfigurationManager on core: most framework APIs will ignore the settings.  It's really just kept for apps to use.

Reply:
It was the simplest path to getting everything to compile; looks like it was related only to your https://github.com/microsoft/msbuild/pull/4638#discussion_r315460218. I'll remove it over there.